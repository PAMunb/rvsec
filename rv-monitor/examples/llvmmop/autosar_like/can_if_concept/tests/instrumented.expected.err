Test 1
Warning: CAN layer was used without proper controller initialization!
Test 2
Test 3
Warning: CAN layer was used without proper controller initialization!
