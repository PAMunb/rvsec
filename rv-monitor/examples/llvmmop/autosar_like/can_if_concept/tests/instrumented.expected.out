Test 1
CanIf component correctly initialized and used.
CanIf component correctly initialized and used.
Test 2
CanIf component correctly initialized and used.
CanIf component correctly initialized and used.
CanIf component correctly initialized and used.
CanIf component correctly initialized and used.
CanIf component correctly initialized and used.
Test 3
